### ISSUE TITLE
HD ts streams h264 on http streams not working after 0.2.6

### ISSUE THREAD

**Issue by kosl** [popcornmix/omxplayer#151]
Similarly to issue #5 after upgrade from omxplayer_0.2.6~git20130427~fcfb7911_armhf.deb to any later version I cannot play live HD streams from my dreambox any more.
Latest sconde version omxplayer_0.3.4~git20140302~16d8b48_armhf.deb plays regular mp4 file but fails with 'Have a nice day' imediately. --live switch doesn't help.

Example 0.2.6 stream that works 

```
$ omxplayer  http://192.168.1.23:31339/0,0074,ffffffff,1121,1120
Aspect : num 64 den 33 aspect 1.777778 pixel aspect 1.939394
Video codec omx-h264 width 528 height 576 profile 100 fps 50.000000
Audio codec mp2 channels 2 samplerate 48000 bitspersample 16
Subtitle count: 0, state: off, index: 1, delay: 0
```

I have upgraded firmware and raspbian and have

```
# vcgencmd version
Feb 28 2014 16:46:25 
Copyright (c) 2012 Broadcom
version a9ba6435869933cdfd1ad5631b6b9a97affd8806 (clean) (release)
```

Failed 0.3.4 debug log looks like

```
22:38:46 T:18446744072351442161   DEBUG: DllBcm: Using omx system library
22:38:46 T:18446744072351447880   DEBUG: DllOMX: Using omx system library
22:38:46 T:18446744072351450809   DEBUG: DllAvFormat: Using libavformat system library
22:38:46 T:18446744072351456059   DEBUG: DBus connection succeeded
22:38:46 T:18446744072351456402   DEBUG: DllAvUtilBase: Using libavutil system library
22:38:46 T:18446744072351456512   DEBUG: DllAvCodec: Using libavcodec system library
22:38:46 T:18446744072351456595   DEBUG: DllAvFormat: Using libavformat system library
22:38:46 T:18446744072351459454   DEBUG: COMXPlayer::OpenFile - avformat_open_input http://192.168.1.23:31339/0,0074,ffffffff,1121,1120
22:38:47 T:18446744072351926359   DEBUG: COMXPlayer::OpenFile - avformat_open_input enabled SEEKING
22:38:47 T:18446744072351929673   DEBUG: OMXThread::Run - Exited thread with  id -1240239008
22:38:47 T:18446744072351931575   DEBUG: OMXThread::StopThread - Thread stopped
```

Other info upon request.
Best regards,
Leon

**Comment by Ruffio** [comment]
@kosl @popcornmix is this resolved or this still an issue?

**Comment by kosl** [comment]
Unfortunately this still remains unsolved issue. Checked 2 weeks ago on my newly installed 4-core raspberry too.
Can I help with some debugging? omxplayer gives up at startup. I can give you access to the machine where you can type  
  omxplayer  $(wget -qO - http://192.168.1.23/video.m3u)
and test old and new version. You will not see anything except terminal, of course. But still you can debug where new version stops.

Tx,
Leon

**Comment by Ruffio** [comment]
@kosl so is has been working before, as in version 0.2.6? @popcornmix would it be possible to have a look in the changelog to see which commit might have caused it?

**Comment by kosl** [comment]
I believe so. The last binary that works is omxplayer_0.2.6~git20130427~fcfb7911_armhf.deb
The very next binary fails. I can prepare short HD sample.ts on dreambox again to locate the problem.

**Comment by Ruffio** [comment]
@kosl I think that would be cool, but I'm not a developer. I do know that
the generel rule is, that the easier it is to reproduce the higher are the
chances of getting someone to look into it ;-)

**Comment by popcornmix** [comment]
fcfb7911 is Apr 27 2013 and 16d8b48 is Mar 2 2014, so looking at the change log is not going to help much.
@kosl can you build from source? Really this needs bisecting between those two commits. It will require about 7 builds to narrow down when the file stopped playing.

**Comment by kosl** [comment]
Yes, I am capable of working with git, compiling sources and debugging. But I am afraid that that sources will not compile on the raspberry.

**Comment by popcornmix** [comment]
Can you try? I've built omxplayer on raspberry before. It should build okay. Give it a try and report if there is a problem.

You can also cross compile using @skgsergio's build env:
https://github.com/skgsergio/rpi-buildenv
